src/components/Modal/Modal.tsx

```diff
@@ -65,7 +65,7 @@ const Modal: React.FC<ModalProps> = ({
               key={key}
               isOpen={item}
               onDismiss={onDismiss ? onDismiss : onClose}
-              style={{ overflow: 'visible', opacity: styles.opacity }}
+              style={{ overflow: 'visible', opacity: styles.opacity, zIndex: 100 }}
```

Comment:
nit: can we make this configurable through a prop like `backdropZIndex` (or smth similar) and default this value (or even `1000` like bootstrap) ?

Different apps might have a different needs for a `zIndex` value. Thoughts?

Reply:
Yes, we can definitely make this configurable.